profiles/profile_utils.py

```diff
@@ -0,0 +1,18 @@
+import pathlib
+import argparse
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-s", "--sql_file", help="the filename of the sqlite file")
+    parser.add_argument("-b", "--batch", help="string indicating the batch name")
+    parser.add_argument("-p", "--plate_name", help="string indicating the platename")
+    parser.add_argument("-f", "--platemap_file", help="path of platmap identifiers")
+    parser.add_argument("-a", "--barcode_platemap_file", help="path of plate info")
+    parser.add_argument("-m", "--moa_file", help="path of moa/target data storing")
+    parser.add_argument("-j", "--project_name", help="string for the project name")
```

Comment:
IIUC, project_name is not read by profile.py or when you start the pipeline in profiling_pipeline.py. It may be safe to remove this option.

Reply:
nice catch! Thanks for pointing out